src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const API_KEY = '8cac6dec66e09ab439c081b251304443'
+export const ENDPOINT = 'https://api.themoviedb.org/3'
+export const ENDPOINT_DISCOVER = ENDPOINT+'/discover/movie/?api_key='+API_KEY+'&sort_by=vote_count.desc'
+export const ENDPOINT_SEARCH = ENDPOINT+'/search/movie/?api_key='+API_KEY
+export const ENDPOINT_MOVIE = ENDPOINT+'/movie/507086?api_key='+API_KEY+'&append_to_response=videos'
```

Comment:
Currently, endpoint construction involves repetitive concatenation and API key inclusion. I suggest centralizing this process for better management. Also, we can use template literals for better code readability and create more functionality for calling api.